SMRefLocation/schema.json

```diff
@@ -11,21 +11,17 @@
     },
     {
       "properties": {
-	    "type": {
+        "type": {
           "type": "string",
           "description": "Property. NGSI-LD Entity Type. It must be equal to SMRefLocation.",
           "enum": [
             "SMRefLocation"
           ]
         },
-		"locationName": {
-          "type": "string",
-          "description": "Property.  Model:'https://schema.org/Text'. Units:'No unit'. Name of the referenced location."
-        },
-		"location": {
-          "$ref": "http://geojson.org/schema/Point.json"
+        "locationName": {
+          "$ref": "https://smart-data-models.github.io/data-models/common-schema.json#/definitions/Location-Commons"
         },

```

Comment:
use addresslocality instead

Reply:
I am referencing to Location-Commons, which contains address object, which in turn contains addressLocality attribute. Would you care to show me the correct way if this wrong? Thanks.